### ISSUE TITLE
Do not expect servers in LB pool in RCv3 setupClass

### ISSUE THREAD

**Issue by manishtomar** [rackerlabs/otter#878]
As an overall goal to make tests independent by setting up stuff in `setUp` instead of `setupClass`, `AutoscaleRackConnectFixture.setupClass` should remove creating sample group to have nodes in the LB pool before running the test. The idea is to ensure that otter does not touch existing servers. This is a valid concern that can be checked by a separate test. Hence, please remove it from `setupClass` and move it to a separate test.

**Comment by sam-falvo** [comment]
@kivattik OK, I remember what this involves now.  We never got around to it due to other priorities.

One way we could achieve this is to abandon the Cafe tests, and rewrite them for Trial.  This will help ensure that all tests are independent by definition, since they have to be to run in parallel.

Thoughts?

**Comment by wirehead** [comment]
Do we have any real reason to do this sooner?  Seems like we can just save this for later.

**Comment by sam-falvo** [comment]
I can't think of any reason to do this sooner.  I just wanted to get written down some approach towards addressing it.  Do we even care?  If so, how best to proceed?  That sort of thing.  Maybe I wasn't terribly clear on this.

**Comment by kivattik** [comment]
I would vote that this get massively backlogged since we are trying to move away from Cafe development entirely.

**Comment by sam-falvo** [comment]
Done.  I'll close this issue on the basis that we won't fix it; instead, when the time comes that this truly becomes an important issue, we can just port the tests over to the Trial-based environment instead.
